libs/model/src/community/GetMembers.query.ts

```diff
@@ -165,8 +127,14 @@ export function GetMembers(): Query<typeof schemas.GetCommunityMembers> {
         order_direction ?? 'DESC',
       );
 
+      // If search query is present count the number of members in the SQL query itself, Otherwise, pull the total
```

Comment:
I feel this is getting too hard to read, usually better to have two distinct queries (two files) one for each scenario to keep things simple

Reply:
Let's get this in as is for now because it has been open for a while and growth is asking for this. @kurtisassad or @Rotorsoft to create a ticket for handling this and creating a pattern for multifile commands/query handlers.